peer/lifecycle/lifecycle.proto

```diff
@@ -159,6 +159,11 @@ message CheckCommitReadinessArgs {
 // supplied as args.
 message CheckCommitReadinessResult{
     map<string, bool> approvals = 1;
+
+    message Mismatch {
```

Comment:
Minor comment - since this contains a repeated list, should it be called "Mismatches" instead of "Mismatch"?